evalml/tests/dependency_update_check/minimum_core_requirements.txt

```diff
@@ -12,7 +12,7 @@ psutil==5.6.6
 requirements-parser==0.2.0
 shap==0.36.0
 texttable==1.6.2
-woodwork==0.6.0
+woodwork==0.7.0
```

Comment:
0.7.0 has a few bugs that were fixed in 0.7.1, but since both of them had to do with dataframes where a woodwork index was set, they may not be relevant for EvalML and 0.7.0 would be a fine minimum requirement